lib/plugins/encrypted_parameters.rb

```diff
@@ -102,8 +102,10 @@ def find_or_create_kms_key
               Moonshot.config.parameters[@kms_key_parameter_name].set(key_arn)
               s.success "Created a new KMS Key for #{@kms_key_parameter_name.blue}!"
             else
-              key_arn = KmsKey.new(Moonshot.config.parameters[@kms_key_parameter_name].value).arn
+              kms=KmsKey.new(Moonshot.config.parameters[@kms_key_parameter_name].value)
+              key_arn = kms.arn
               s.success "Using existing KMS Key for #{@kms_key_parameter_name.blue}!"
+              KmsKey.new(key_arn).update
```

Comment:
Why do we need to create a new object again here? could it just be kms.update since kms contains the KmsKey object.